### PR TITLE
add asciidoctor-diagram support

### PR DESCRIPTION
**Pull request by oalyman**
Enables the usage of https://asciidoctor.org/docs/asciidoctor-diagram/

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -6,7 +6,7 @@ name := "sbt-site"
 
 organization := "com.typesafe.sbt"
 
-version := "1.3.2"
+version := "1.3.3-SNAPSHOT"
```
Comment (jonas) [inline build.sbt]: :+1:

File: build.sbt [hunk 2]
```diff
@@ -28,7 +28,8 @@ libraryDependencies ++= Seq(
   "org.foundweekends" %% "pamflet-library" % "0.7.1",
   "org.yaml"        % "snakeyaml"        % "1.13",
   "com.typesafe"    % "config"           % "1.2.1", // Last version to support Java 1.6
-  "org.asciidoctor" % "asciidoctorj"     % "1.5.4"
+  "org.asciidoctor" % "asciidoctorj"     % "1.5.4",
+  "org.asciidoctor" % "asciidoctorj-diagram" % "1.5.4.1"
```
Comment (jonas) [inline build.sbt]: Why is this using a different version?

**Comment by jonas** [comment]
Fixes #76 it seems

File: build.sbt (hunk 2 above)
Comment (oalyman) [reply build.sbt]: The different version wasn't intentional.

Should I downgrade the asciidoctorj-diagram dependency to 1.5.4, or upgrade the asciidoctorj dependecy to 1.5.4.1?

1.5.4.1 is a bugfix release https://github.com/asciidoctor/asciidoctorj/releases/tag/v1.5.4.1

**Comment by jonas** [comment]
Thanks again.

**Comment by smrutimo** [comment]
@oalyman Any idea when this version 1.3.3 gets released ?

**Comment by oalyman** [comment]
@smrutimo no, maybe @jonas knows.

**Pull request merged**